src/views/day/loop.php

```diff
@@ -0,0 +1,113 @@
+$today             = $php_class->today();
+$container_class   = $php_class->array_of_container_classes_based_on_today_to_string();
+
+$all_day_text = Tribe__Extension__Schedule_Day_View::instance()->get_all_day_text();
+
+$now = Tribe__Extension__Schedule_Day_View::instance()->now_timestamp();
+?>
+
+<div
+	id="tribe-events-day"
+	class="tribe-events-loop<?php echo esc_attr( $container_class ); ?>"
+	data-tribe-timezone="<?php echo esc_attr( Tribe__Events__Timezones::wp_timezone_string() ); ?>"
+	data-tribe-now="<?php echo esc_attr( $now ); ?>"
+>
+
+	<?php
+	// Used by Tribe__Extension__Schedule_Day_View::setup_loop()
+	do_action( $prefix . '_inside_before_loop' );
+	$all_time_slots = array_merge( array( $all_day_text ), array_keys( $wp_query->get( 'time_slots' ) ) );
+
+	foreach ( $all_time_slots as $current_time_slot ) :
+		Tribe__Extension__Schedule_Day_View::instance()->build_current_time_slot_args( $current_time_slot );
+		$current_time_slot_args = Tribe__Extension__Schedule_Day_View::instance()->current_time_slot_args;
+
+		$active_class = '';
+		if ( $today ) {
+			if (
+				$current_time_slot_args['is_all_day_time_slot']
+				|| (
+					$now >= $current_time_slot_args['start_timestamp']
+					&& $now <= $current_time_slot_args['end_timestamp']
+				)
+			) {
+				$active_class = ' tribe-events-day-grouping-is-active tribe-events-day-grouping-is-now';
+			}
+		} elseif ( ! empty( $current_time_slot_args['time_slot_event_count'] ) ) {
+			$active_class = ' tribe-events-day-grouping-is-active';
+		}
+		?>
+
+		<div
+			id="<?php echo esc_attr( $current_time_slot_args['time_slot_id'] ); ?>"
+			class="tribe-events-day-time-slot<?php echo esc_attr( $active_class ); ?>"
+			data-tribe-groupstart="<?php echo esc_attr( $current_time_slot_args['start_timestamp'] ); ?>"
+			data-tribe-groupend="<?php echo esc_attr( $current_time_slot_args['end_timestamp'] ); ?>"
+			data-tribe-groupeventscount="<?php echo esc_attr( $current_time_slot_args['time_slot_event_count'] ); ?>"
+			<?php
+			if ( ! empty( $current_time_slot_args['time_slot_event_count'] ) ) {
+				echo 'data-tribe-grouphasevents';
+			} else {
+				echo 'data-tribe-grouphasnoevents';
+			}
+			?>
+		>
+			<h5>
+				<button
+					id="<?php echo esc_attr( $current_time_slot_args['button_id'] ); ?>"
+					class="tribe-events-day-group-trigger"
+					aria-expanded="<?php echo esc_attr( $current_time_slot_args['aria_expanded_on_load'] ); ?>"
+					<?php
+					if ( empty( $current_time_slot_args['time_slot_event_count'] ) ) {
+						echo 'disabled';
+					}
+					?>
+				>
+					<?php echo esc_html( $current_time_slot_args['time_slot_title'] ); ?>
+					<?php
+					if ( ! empty( $current_time_slot_args['time_slot_event_count'] ) ) {
+						printf( '<span>%s</span>', esc_html__( "Toggle Group's Events", 'tribe-ext-schedule-day-view' ) );
+					}
+					?>
+				</button>
+			</h5>
+			<?php
+			while ( have_posts() ) : the_post();
```

Comment:
Avoid multiple statements on a single line

Reply:
We're not mixing PHP and HTML here, so no need to use the alternative (ie non-curly brace) syntax

Reply:
done